lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.spec.ts

```diff
@@ -503,7 +503,7 @@ describe('TaskHeaderCloudComponent', () => {
             const createdDateElement = fixture.debugElement.query(By.css('[data-automation-id="header-created"] .adf-property-value'));
 
             expect(component.dateFormat).toEqual('full');
-            expect(createdDateElement.nativeElement.innerText.trim()).toBe('Monday, December 17, 2018 at 12:00:55 PM GMT+00:00');
+            expect(createdDateElement.nativeElement.innerText.trim()).toBe('Monday, December 17, 2018 at 12:00:00 AM GMT+00:00');
```

Comment:
I had to change the expectation due to the zero time when dealing with date-only types